[docs] Explain to always use `--base_dir` on yugabyted commands when not using default value (#26908)
yugabyte/yugabyte-db: docs/content/preview/tutorials/quick-start/docker.md

  If running macOS Monterey, replace `-p7000:7000` with `-p7001:7000`.

{{< note title="Note" >}}

> When you start `yugabyted` with a non-default `--base_dir` value, you will have to set that on every command to `yugabyted`.

Reviewer: ```suggestion
When you start yugabyted with a non-default `--base_dir` value, you must set the `--base_dir` flag for every subsequent yugabyted command.
```

> For example: to get the status of the cluster created above, you'd use `bin/yugabyted status --base_dir=/home/yugabyte/yb_data`.

Reviewer: ```suggestion
For example, to get the status of the cluster you just created, you would enter `bin/yugabyted status --base_dir=/home/yugabyte/yb_data`.
```
